apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/cache/CacheConfigurator.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c)  2018-2019. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.util.cache;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+
+import java.util.Optional;
+
+public class CacheConfigurator<K, V> implements CacheConfiguration<K, V> {
+    private String name;
+    private long elementSize;
+    private int cachePriority;
+    private int maxSize = -1;
+    private CacheBuilder cacheBuilder;
+    private Optional<CacheLoader<K, V>> optionalCacheLoader;
+
+    public CacheConfigurator(String name, long elementSize, int cachePriority) {
+        this(name, elementSize, cachePriority, null);
+    }
+
+    public CacheConfigurator(String name, long elementSize, int cachePriority, CacheLoader<K, V> cacheLoader) {
+        this.name = name;
```

Comment:
Andrew added NULL checks here for 'name' values yesterday, as far as I remember.